portfolio/src/global.d.ts

```diff
@@ -0,0 +1,2 @@
+declare module "*.png";
```

Comment:
This makes Typescript happy so that we can just import the image sources directly into our components instead of having to reference them via an asset path in the `src` property of the image tags. This is just another typescript config file that you can kind of ignore -- no need to fully understand exactly what's going on here